sample_project/Source/sample_project/Feature Layer/FeatureLayer.h

```diff
@@ -21,34 +21,30 @@
 #include "ArcGISSamples/Public/ArcGISPawn.h"
 #include "FeatureLayer.generated.h"
 
-UCLASS(Blueprintable)
-class SAMPLE_PROJECT_API UWebLink : public UObject
+USTRUCT(BlueprintType)
+struct SAMPLE_PROJECT_API FWebLink
 {
 	GENERATED_BODY()
 public:
-	UPROPERTY(BlueprintReadWrite, VisibleAnywhere)
+	UPROPERTY(BlueprintReadWrite, EditAnywhere)
 	FString link;
-	UPROPERTY(BlueprintReadWrite, VisibleAnywhere)
+	UPROPERTY(BlueprintReadWrite, EditAnywhere)
 	FString requestHeaders;
-	UPROPERTY(BlueprintReadWrite, VisibleAnywhere)
+	UPROPERTY(BlueprintReadWrite, EditAnywhere)
+	TArray<FString> outFields;
+	FString headers;
 	FString outFieldHeader;
 };
 
 USTRUCT(BlueprintType)
-struct SAMPLE_PROJECT_API FFeature
+struct SAMPLE_PROJECT_API FProperties
```

Comment:
`FProperties` might be too generic.
```suggestion
struct SAMPLE_PROJECT_API FFeatureLayerProperties
```